lib/infrastructure/settings_repository.dart

```diff
@@ -11,13 +13,17 @@ const _defaults = {_keyIsSignupEnabled: false};
 const _refreshInterval = Duration(minutes: 5);
 const _fetchTimeout = Duration(seconds: 60);
 final _minimumFetchInterval = _fetchTimeout + const Duration(seconds: 1);
+const _releaseBaseApiEndpoint = 'http://api.collaction.org';
```

Comment:
We can consider using dotenv for environment variables, the API will be running locally once we get it up and running (versioned), and this will help separating dev and prod environment.

Reply:
So you only want to configure the endpoint at compile time?
I think it might be more convenient to configure it at runtime using test ui on debug builds as well.